codec/src/main/java/io/netty/handler/codec/ByteToMessageDecoder.java

```diff
@@ -522,8 +517,9 @@ protected void decodeLast(ChannelHandlerContext ctx, ByteBuf in, List<Object> ou
         }
     }
 
-    static ByteBuf expandCumulation(ByteBufAllocator alloc, ByteBuf oldCumulation, ByteBuf in) {
```

Comment:
Might be best to keep this package-private - I expect it may be intentional to avoid a synthetic method since it is called from the cumulator classes. Of course adding a comment to make that clear wouldn't harm!

Reply:
Yes we were keen on avoiding synthetic generation in the past ... however IIRC JDK11+ avoids this now anyways. @normanmaurer dropped a comment on the previous iteration of this PR to make it private, but happy to keep it package private if there is strong need.

Reply:
Whatever you guys prefer...

Reply:
I will revert in this PR as this change isn't necessary in this context.